pages/emotion-records.js

```diff
@@ -0,0 +1,36 @@
+import EmotionRecordsList from "@/components/EmotionRecordsList";
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: black;
+  margin: 1rem;
+  padding: 1rem;
+  border-radius: 8.5px;
+  &:hover {
+    background-color: lightskyblue;
+  }
```

Comment:
Please keep in mind: The `:hover` style will not have any effect on touch devices, as they do not support hover states. Consider alternative styles or interactions for mobile and touch interfaces to enhance the user experience on these devices.